### ISSUE TITLE
Remove normative statements from vocab

### ISSUE THREAD

**Issue by awoie** [w3c/vc-data-model#1077]
IMO, the vocab (https://www.w3.org/2018/credentials/) should not contain normative statements (verbs) about things the VCDM already states. This is confusing to readers and redundant to the normative statements made in the VCDM specification (=> aka THE "standard").

I suggest removing all normative verbs from the vocab. Some examples are listed below ...

> The value of the holder property MUST be a URI. It is RECOMMENDED that dereferencing the URI results in a document containing machine-readable information about the holder that can be used to verify the information expressed in the credential.

> The value of the issuer property MUST be a URI. It is RECOMMENDED that dereferencing the URI results in a document containing machine-readable information about the issuer that can be used to verify the information expressed in the credential.

**Comment by awoie** [comment]
For example, the Activity Pub work (also W3C REC) does not use any normative verbs in their vocab: https://www.w3.org/ns/activitystreams

**Comment by andyfmiller** [comment]
> This is confusing to readers and redundant to the normative statements made in the VCDM specification

And conflicting in at least one place. As you noted the vocab describes Issuer as:

> The value of the issuer property MUST be a URI.

While the standard describes it:

> The value of the issuer [property](https://www.w3.org/TR/vc-data-model-2.0/#dfn-property) MUST be either a [URL](https://www.w3.org/TR/vc-data-model-2.0/#dfn-url) or an object containing an id [property](https://www.w3.org/TR/vc-data-model-2.0/#dfn-property).

**Comment by iherman** [comment]
I agree. The separate, but relevant, issue on this is #1080: once the VCDM is stable and all terms have a proper anchor, the vocabulary specification should be systematically changed, removing most of the texts and referring to the VCDM specification itself as a normative source of the description. It is on my desk to do so, but #1080 should be completed beforehand.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-06-21-vcwg#section2-2) on 2023-06-21

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 2.2. Remove normative statements from vocab (issue vc-data-model#1077)

_See github issue [vc-data-model#1077](https://github.com/w3c/vc-data-model/issues/1077)._




**Kristina Yasuda:** We don't have ivan on this call. Do we mark as before CR?  

> *Brent Zundel:* +1.

**Manu Sporny:** Could we mark it as pending close instead? 1080 is already saying that Ivan wants to anchor the things in the vocab to the specification. If we do 1080, then 1077 gets closed.  
… Could we mark as duplicate?  

**Brent Zundel:** My understanding is that they are different. One talks about adding statements to the vocab. 1077 mentions what to do with the ones that are already normative.  

> *Manu Sporny:* I wonder if Ivan could check if 1080 and 1077 are dupes of one another and they can be closed?

</details>

**Comment by msporny** [comment]
This issue is very close to a duplicate of #1080. Based on comments from @iherman here: https://github.com/w3c/vc-data-model/issues/1080#issuecomment-1602155809 , I'm marking this issue as a duplicate and pending close.

**Comment by brentzundel** [comment]
No objections raised since marked `pending close`, closing.
